### DISCUSSION TITLE
How do I change the update prompt to Chinese

### DISCUSSION THREAD

**Discussion by chinapostm** [tauri-apps/tauri#7187] [Q&A]
![20230613041643](https://github.com/tauri-apps/tauri/assets/75209261/cdcda91f-0e29-4060-a324-122e165e6052)

**Comment by FabianLars** [comment]
You can't, the text is hardcoded. You'll have to disable the default dialog and create your own dialog from scratch. An introduction showing how to use the updater apis can be found here: https://tauri.app/v1/guides/distribution/updater#custom-dialog
